burnman/eos/slb.py

```diff
@@ -70,18 +104,13 @@ def _thermal_pressure(self,T,V, params):
         gr = self.grueneisen_parameter(0., T, V, params) # P not important
         P_th = gr * debye.thermal_energy(T,Debye_T, params['n'])/V
         return P_th
-    
+
     def _delta_pressure(self,x,pressure,temperature,V_0,T_0,Debye_0,n,a1_ii,a2_iikk,b_iikk,b_iikkmm):
```

Comment:
Can you describe why it is a performance improvement to split up the function this way and call `_delta_pressure_fast1` and `delta_pressure_fast2`? Can one jitted function not call another jitted function?